tools/test-proxy/Azure.Sdk.Tools.TestProxy.Tests/PlaybackTests.cs

```diff
@@ -248,5 +250,97 @@ public async Task TestPlaybackWithGZippedContentPlayback()
             HttpResponse response = new DefaultHttpContext().Response;
             await testRecordingHandler.HandlePlaybackRequest(recordingId, request, response);
         }
+
+
+        [Theory]
+        [InlineData(
+        @"{
+              ""AssetsRepo"": ""Azure/azure-sdk-assets"",
+              ""AssetsRepoPrefixPath"": """",
+              ""AssetsRepoId"": """",
+              ""TagPrefix"": ""python/tables"",
+              ""Tag"": ""python/tables_0f2a2fa5""
+        }")]
+        [Trait("Category", "Integration")]
+        public async Task TestPlaybackWithRelativeAssetsJsonPath(string inputJson)
+        {
+            var folderStructure = new string[]
+            {
+                Path.Join("sdk", "tables", GitStoretests.AssetsJson)
+            };
+
+            Assets assets = JsonSerializer.Deserialize<Assets>(inputJson);
+            var testFolder = TestHelpers.DescribeTestFolder(assets, folderStructure);
+
+            try
+            {
+                RecordingHandler testRecordingHandler = new RecordingHandler(testFolder, new GitStore());
+                var httpContext = new DefaultHttpContext();
+                var recordingPath = "sdk/tables/azure-data-tables/tests/recordings/test_challenge_auth.pyTestTableChallengeAuthtest_challenge_auth_supported_version.json";
+                var body = "{\"x-recording-file\":\"" + recordingPath + "\", \"x-recording-assets-file\": \"sdk/tables/assets.json\"}";
+                httpContext.Request.Body = TestHelpers.GenerateStreamRequestBody(body);
+                httpContext.Request.ContentLength = httpContext.Request.Body.Length;
+
+                var controller = new Playback(testRecordingHandler, new NullLoggerFactory())
+                {
+                    ControllerContext = new ControllerContext()
+                    {
+                        HttpContext = httpContext
+                    }
+                };
+                await controller.Start();
```

Comment:
This looks like we're not actually testing anything. We are.

The difference is that we have passed in the assetsjson path to /playback/start. This will auto-restore the targeted tag and attempt to load the recording file from that new context. If the file wasn't present, the recording would not load.